buildkite/bazelci.py

```diff
@@ -2900,7 +2900,7 @@ def print_bazel_downstream_pipeline(
     if (
         not test_disabled_projects
         and not test_incompatible_flags
-        and os.getenv("BUILDKITE_BRANCH") == "master"
+        and os.getenv("BUILDKITE_BRANCH") in ("master", "stable", "main")
```

Comment:
I'd prefer to keep it this way for now - I personally find it easier to read code like this rather than get lost in a deep call tree of short functions...

This is borderline case - we already have something like "IsPullRequest" IIRC, so it would fit into that, but we only do this particular check here two times, so.. mhm.. let's just keep it like this for now.

Reply:
I was wrong - it's three times after my change 😆 I extracted it into a `current_branch_is_main_branch` method now, that looks pretty nice.